cypress/integration/admin_portal/tests/test_code_management.spec.js

```diff
@@ -0,0 +1,53 @@
+import LandingPage from '../pages/landing_page'
+import CodeManagementPage from '../pages/enterprise_code_management'
+
+describe('landing page tests', function () {
+  const landingPage = new LandingPage()
+  const codeManagementDashboard = new CodeManagementPage()
+
+  before(function(){
+    cy.login_using_api(Cypress.env('ADMIN_PORTAL_USER_EMAIL'), Cypress.env('ADMIN_PORTAL_USER_PASSWORD'))
+  })
+
+  beforeEach(function () {
+    Cypress.Cookies.preserveOnce('edxloggedin', 'stage-edx-user-info', 'stage-edx-sessionid')
+    cy.visit('/')
+    landingPage.goToEnterprise('SuccessFactors')
+    landingPage.openCodeManagement()
+  })
+
+  it('Verifies the validation checks on REQUEST MORE CODES form', function () {
+   codeManagementDashboard.requestMoreCodes()
+   codeManagementDashboard.getFormField('emailAddress').should('have.attr', 'value', 'zrana@edx.org')
+   codeManagementDashboard.getFormField('enterpriseName').should('have.attr', 'value', 'SuccessFactors')
+   codeManagementDashboard.getLabels('Company').should('have.text', 'Company*').children().should('have.class', 'required')
+   codeManagementDashboard.getLabels('Email').should('have.text', 'Email Address*').children().should('have.class', 'required')
+   codeManagementDashboard.getLabels('Number of Codes').should('have.text', 'Number of Codes')
+   codeManagementDashboard.getRequestCodesButton().should('have.attr', 'type', 'submit').should('have.text', 'Request Codes')
```

Comment:
it would look nicer if you use in place of 2nd should
https://docs.cypress.io/api/commands/and.html#Syntax